### ISSUE TITLE
pyotdr is not correct

### ISSUE THREAD

**Issue by yangyh33** [sid5432/pyOTDR#36]
from pyOTDR.dump import tofile, ExportDataType
from pyOTDR.read import sorparse

**Comment by sid5432** [comment]
Can you be more specific about what error you are seeing? Please keep in mind that this only works with Python 3; the code no longer works under Python 2 (I know, the README file still says Python 2). If you are seeing some warning about syntax error in dump, that's because it is using Python 3 features not support in Python 2.

**Comment by RemiDesgrange** [comment]
pypi.org is right and the badge in the readme is right, but some text is not. I'll correct this asap !

Please, @yangyh33 be more specific about your setup, what error do you get, what's your environment.

**Comment by RemiDesgrange** [comment]
At least #37 README is fixed :-)

**Comment by yangyh33** [comment]
When I run main.py, The shell's promption:

Python 3.8.2 (tags/v3.8.2:7b3ab59, Feb 25 2020, 22:45:29) [MSC v.1916 32 bit (Intel)] on win32
Type "help", "copyright", "credits" or "license()" for more information.
>>> 
======== RESTART: D:\Download\pyOTDR-master\pyOTDR-master\pyotdr\main.py =======
Traceback (most recent call last):
  File "D:\Download\pyOTDR-master\pyOTDR-master\pyotdr\main.py", line 4, in <module>
    from pyotdr.dump import tofile, ExportDataType
ModuleNotFoundError: No module named 'pyotdr'

Because Python is case sensitive, pyotdr should be "pyOTDR". Is it right?

**Comment by RemiDesgrange** [comment]
How did you install `pyOTDR` ?

**Comment by yangyh33** [comment]
pip install pyotdr

**Comment by RemiDesgrange** [comment]
This is weird, on my machine (Linux) here is a step I used to have a working env:

```
mkdir /tmp/test-pyotdr-env && cd /tmp/test-pyotdr-env
python -m venv .venv
source .venv/bin/activate
pip install pyotdr
python
>>> from pyotdr.dump import tofile # working on my env
>>> exit()
deactivate
```

I don't know how is your setup. Is pyotdr present in the `sites-packages` folder?

**Comment by yangyh33** [comment]
![image](https://user-images.githubusercontent.com/87868767/127140245-a810bcdb-9852-4f6c-a240-b5ed8dc153ca.png)

**Comment by RemiDesgrange** [comment]
Then this is not a pyotdr problem, but a problem with your environment. Try checking your python path like this to find out why your python interpreter cannot find pyotdr

```
>>>import sys
>>>sys.path
```

**Comment by yangyh33** [comment]
I mean the interpreter can cope with this:
from pyOTDR.dump import tofile, ExportDataType
from pyOTDR.read import sorparse

But when the code like this, the generator will generate error promption.
from pyotdr.dump import tofile, ExportDataType
from pyotdr.read import sorparse

**Comment by RemiDesgrange** [comment]
So? what's the problem then? :sweat_smile:

**Comment by RemiDesgrange** [comment]
Ok, so @sid5432 looked to your problem and you were totally right. There is an issue with windows.

We are discussing the process of modifying  the code so that calling `import pyotdr` would work on any platform, but this would imply a breaking change (I think)

For now just call 

```
from pyOTDR.dump import tofile, ExportDataType
from pyOTDR.read import sorparse
```

**Comment by sid5432** [comment]
To clarify a few points: in Windows, the pip installation gives you an executable (pyOTDR.exe) in the Scripts folder, and a pyOTDR folder in the site-packages folder. In the the main.py file in the site-packages/pyOTDR folder, the import statement is "pyotdr" instead of "pyOTDR". But if you are using the pyOTDR.exe binary, just changing "pyOTDR" to "pyotdr" in main.py does not fix the problem. It seems the binary already has the "from pyOTDR.dump ..." statements baked in.

As a temporary fix, rename the pyOTDR folder to pyotdr in the site-packages folder. But I haven't tested this thoroughly, so there could be other problems.

**Comment by RemiDesgrange** [comment]
Fix in #38
